docs/meetings/2021/TechArea20210223.md

```diff
@@ -54,19 +54,16 @@ In the HTCondor-CE Gratia Probe ProbeConfig, the data folder should match the pe
 ### Support Update
 
 -   SIUE (Southern Illinois University Edwardsville) (BrianL): got successful Ubuntu 20 pilots with patches to HTCondor
+-   Purdue (Carl, Derek): Gratia probe for 
```

Comment:
@djw8605 looks like an unfinished comment? This was the issue where they need to run a custom probe, right?